### PR TITLE
Bump ipython 3.2.3 and use intel-2015b and PyZMQ 15.1.0

### PR DESCRIPTION
**Pull request by ehiggs**

### CODE REVIEW

File: easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    (python, pyver),
+    ('PyZMQ', '15.1.0', '%s-zmq4' % versionsuffix),
+]
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+exts_list = [
+    ('pysqlite', '2.7.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb]: use 2.8.1?

File: easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        'modulename': 'pysqlite2',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pysqlite/'],
+    }),
+    ('requests', '2.7.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb]: 2.9.1?

File: easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests/'],
+    }),
+    ('Pygments', '2.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/P/Pygments/'],
+    }),
+    ('tornado', '4.2.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb]: 4.3?

File: easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        'source_urls': ['https://pypi.python.org/packages/source/t/tornado/'],
+    }),
+    ('Jinja2', '2.8', {
+        'source_urls': ['https://pypi.python.org/packages/source/J/Jinja2/'],
+    }),
+    ('jsonschema', '2.5.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/j/jsonschema/'],
+    }),
+    ('mistune', '0.7', {
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb]: 0.7.1

File: easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        'source_urls': ['https://pypi.python.org/packages/source/m/mistune/'],
+    }),
+    ('ptyprocess', '0.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/ptyprocess/'],
+    }),
+    ('terminado', '0.5', {
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb]: 0.6

File: easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+sanity_check_commands = [
+    ('ipython -h', ''),
+    ('ipython notebook --help', ''),
+    ('iptest', ''),
+]
+
+moduleclass = 'tools'
+
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb]: drop empty line pls

**Comment by ehiggs** [comment]
Updated the versions. Please re-review/test.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/78cf8cfd94517461b2ce for a full test report.

File: easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+        'modulename': 'pysqlite2',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pysqlite/'],
+    }),
+    ('requests', '2.9.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests/'],
+    }),
+    ('Pygments', '2.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/P/Pygments/'],
+    }),
+    ('tornado', '4.3', {
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb]: this requires `backports-abc`, please include it as an extra extension above this one

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/328f365a45d26b4c0773 for a full test report.

File: easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    ('requests', '2.9.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests/'],
+    }),
+    ('Pygments', '2.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/P/Pygments/'],
+    }),
+    ('backports_abc', '0.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/backports_abc/'],
+    }),
+    ('tornado', '4.3', {
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-3.2.3-intel-2015b-Python-2.7.10.eb]: another missing dep of tornado `certifi` (2015.11.20.1)

based on https://github.com/tornadoweb/tornado/blob/master/setup.py, these may also need to be added: `singledispatch` and `backports.ssl_match_hostname`

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 5 out of 6 (1 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8fea8358bb4767310ed8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c959cd7e0496ddd3781a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ehiggs!

**Comment by boegel** [comment]
Going in, thanks @ehiggs!

**Comment by ehiggs** [comment]
Thanks @boegel

**Pull request merged**